interview-questions.md

```diff
@@ -7,53 +7,54 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is a block in Ruby?
 
-  Your answer:
+  Your answer: Ruby blocks are functions that can be passed into a methods. Blocks dont have names.
 
-  Researched answer:
+  Researched answer:In short, blocks are unnamed methods. They are passed to methods just like other objects. Blocks consist of code between braces or a do/end pair. Block become helpful when using shared logic to various context.
 
 
 
 2. What is a gem?
 
-  Your answer:
+  Your answer: A gem is a Ruby package that contains a library. They help functionality of Ruby applications.
 
-  Researched answer:
+  Researched answer: Ruby gems are basically open source libraries that contain Ruby code and are sprinkled with a little extra data. These gems allow us to use some of the code within the gem, without explicitly inserting that code. (https://rubygems.org/.) Awesome, so there are a plethora of gems to explore with a multitube of purposes. Here's a couple of basics. Bundler, which provides a consistent environment for Ruby project by tracking and installing the ideal gems and versions that are needed. #1 downloaded gem of all time. RSpec, is a testing framework for Ruby. Devise is a authentication for user logins. Also, Rails is a gem! Rails provides the fullstack framework that allows Ruby to be transformed into web applications. Appearntly Ruby is basically a 'one-stop-shop' for web dev. 
```

Comment:
Great answer!  We will be getting into Devise in a few weeks for authentication and definitely be using the versatile abilities of Ruby on Rails to create full-stack applications in the near future.